libraries/persistent_store/src/storage.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Represents a byte position in a storage
```

Comment:
Comment style was usually full sentences with dots at the end.

Reply:
Right, I removed because the first line is obviously a single "sentence" and it adds a bit of reading clarity. But I can add back. Both styles are widespread on the web.

Reply:
Oh I didn't get notification for the reaction on original post. Since it's 2 votes, I did the change back.